lib/Git/Repository/Plugin.pm

```diff
@@ -17,15 +16,16 @@ sub install {
     } @keywords;
     carp "No keywords installed from $class" if !@keywords;
 
-    # install keywords
-    *{"Git::Repository::$_"} = \&{"$class\::$_"} for @keywords;
+    for (@keywords) {
+        no strict 'refs';
+        *{"Git::Repository::$_"} = \&{"$class\::$_"};
+    }
 }
 
 sub _keywords {
     my ($class) = @_;
-    no strict 'refs';
     carp "Use of \@KEYWORDS by $class is deprecated";
-    return @{"$class\::KEYWORDS"};
+    return do { no strict 'refs'; @{"$class\::KEYWORDS"} };
```

Comment:
This is annoying to read. Sometimes it’s necessary but in this case you can just move the `no strict 'refs'` down one line.